addons/mail/models/mail_mail.py

```diff
@@ -78,15 +78,19 @@ def default_get(self, fields):
     auto_delete = fields.Boolean(
         'Auto Delete',
         help="This option permanently removes any track of email after it's been sent, including from the Technical menu in the Settings, in order to preserve storage space of your Odoo database.")
-    scheduled_date = fields.Char('Scheduled Send Date',
+    scheduled_date = fields.Datetime('Scheduled Send Date',
```

Comment:
As you change the DB schema, could you update mail ``__manifest__`` version to 1.9 ?